### PR TITLE
Bugfix/add distractor analysis green background

### PR DESCRIPTION
**Pull request by wtritch**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/styles.less [hunk 1]
```diff
@@ -915,3 +915,9 @@ body .ui-widget.ui-datatable table .ui-datatable-data > tr > td.level-down,
 a.btn.disabled.with-pointer {
   pointer-events: auto;
 }
+
+//TODO move to SBAC UI kit
+distractor-analysis .ui-widget.ui-datatable table .ui-datatable-data td.green {
+  color: contrast(@green, darken(@green, 100%), lighten(@green, 100%));
+  background-color: @green;
+}
```
Comment (wtritch) [inline webapp/src/main/webapp/src/styles.less]: I'm using the same process as the SBAC UI kit for contrasting background with text color.

**Comment by malaffoon** [comment]
Just curious: how did this break?

File: webapp/src/main/webapp/src/styles.less (hunk 1 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/styles.less]: 👍 we'll need to for-loop this through all the `@colorVars` in the kit

**Comment by wtritch** [comment]
Not sure.  Likely as a result of some style refactoring?

**Comment by esotrope** [comment]
@malaffoon i think i accidentally removed the style with the intention to relocate it but never repasted it anywhere x l

File: webapp/src/main/webapp/src/styles.less (hunk 1 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/styles.less]: Yup, it belongs in the SBAC UI kit in mixins.less eventually.

**Comment by malaffoon** [comment]
Okay thanks. Just trying to understand if this was something like that (no biggie) or some systemic style issue (which would suck).

**Pull request merged**